Catalog/Examples/Catalog/CatalogViewController.swift

```diff
@@ -19,7 +19,7 @@ final class CatalogViewController: UIViewController {
     
     private lazy var _tableView: UITableView = _createTableView()
     
-    private lazy var _dataSourceDelegate: TableDataSourceDelegate = {
+    private lazy var _tableDataSource: TableDataSourceDelegate = {
```

Comment:
I see you removed `_` in the presenter above, do we need to remove them here as well?

Reply:
For some reason swiftlint didn't scream here, but to align with the new coding style I'll remove them everywhere 😄